src/models/animals.js

```diff
@@ -16,19 +16,20 @@ const AnimalSchema = new Schema({
     type: String,
     required: true
   },
-  birthday: {
+  dateOfBirth: {
```

Comment:
Changed to reflect the actual date of birth (from which we can extract the birthday if we want to)